src/VaultSecretClient.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace SublimeSkinz\SublimeVault;
+
+use GuzzleHttp\Exception\GuzzleException;
+use SublimeSkinz\SublimeVault\VaultClientFactory;
+use Analog\Logger;
+use Analog\Handler\File;
+
+class VaultSecretClient
+{
+
+    protected $vault;
+    protected $logger;
+
+    public function __construct()
+    {
+        $this->vault = VaultClientFactory::create(getenv('VAULT_ADDR'), getenv('VAULT_AUTH_METHOD'), getenv("VAULT_BUCKET_NAME"), getenv("VAULT_CREDS_PATH"));
+
+        $logger = new Logger();
+        $logger->handler(File::init(__DIR__ . '/../logs/errors.log'));
```

Comment:
Same for the log here also it will be great to let manage if itś raw log or json log or ...